src/com/yetanalytics/lrs_admin_ui/functions/http.cljs

```diff
@@ -6,13 +6,16 @@
             goog.string.format))
 
 (defn serv-uri
-  [server-host path]
-  (str server-host path))
+  [server-host path & {:keys [proxy-path]
+                       :or {proxy-path ""}}]
+  (str server-host proxy-path path))
 
 (defn build-xapi-url
-  [server-host xapi-prefix path params]
+  [server-host xapi-prefix path params  & {:keys [proxy-path]
+                                           :or {proxy-path ""}}]
   (let [path' (or path
-                  (format "%s/statements"
+                  (format "%s%s/statements"
+                          (if (some? proxy-path) proxy-path "")
```

Comment:
Wouldn't `(some? proxy-path)` always be `true` since you're setting it to `""` as the default in the kwargs?

(Also there's an extra space before the kwarg `&`.)

Reply:
I was actually just trying to figure this out, its something to do with `format`. Without this it keeps injecting null into my paths and this stops it. I will figure it out fully before I change.

Reply:
4b1b91c32b43cc10aa2d3d44fba800b69e59914b